packages/glibc.rb

```diff
@@ -3,20 +3,21 @@
 class Glibc < Package
   description 'The GNU C Library project provides the core libraries for GNU/Linux systems.'
   homepage 'https://www.gnu.org/software/libc/'
-  version LIBC_VERSION
   license 'LGPL-2.1+, BSD, HPND, ISC, inner-net, rc, and PCRE'
-  compatibility 'all'
```

Comment:
I think you can leave these lines.

Reply:
OK, I think we can keep the version down in the case block because 2.32-1, although I think we could get away with not incrementing since it hasn't hit the stable branch yet.